scripts/start.sh

```diff
@@ -8,26 +8,11 @@ echo 'Enter y or n'
 
 read response
 
-if [[ $response == "n" ]]; then
-  if [ -d "venv" ] ; then
-    source venv/bin/activate
-  else
-    echo -e '\n\n\033[31mYou do not have a venv folder!\033[0m\n'
-  fi
+if [ $response == "n" ]; then
+  source venv/bin/activate
 else
   echo "Virtualenv Wrapper in use!"
 fi
 
-function start_client() {
```

Comment:
Why are we deprecating this? Does this work outside docker? Also, how are you calling this script?

`scripts/start.sh` is how it is called via the readme. Will this still work if called like that, I just need some context here